scripts/mina-local-network/monitor_memory.py

```diff
@@ -0,0 +1,165 @@
+import collections
+import os
+import re
```

Comment:
Let's introduce this script as a separate PR, it's not directly related to this PR